Makefile

```diff
@@ -210,12 +210,12 @@ SPHINX_HTML_OUTDIR:=dist-newstyle/doc/users-guide
 USERGUIDE_STAMP:=$(SPHINX_HTML_OUTDIR)/index.html
 
 # do pip install every time so we have up to date requirements when we build
-users-guide: .python-sphinx-virtualenv $(USERGUIDE_STAMP)
+users-guide: .python-sphinx-virtualenv/bin/activate $(USERGUIDE_STAMP)
 $(USERGUIDE_STAMP) : doc/*.rst
 	mkdir -p $(SPHINX_HTML_OUTDIR)
 	(. ./.python-sphinx-virtualenv/bin/activate && pip install -r doc/requirements.txt && $(SPHINXCMD) $(SPHINX_FLAGS) doc $(SPHINX_HTML_OUTDIR))
 
-.python-sphinx-virtualenv:
+.python-sphinx-virtualenv/bin/activate:
 	python3 -m venv .python-sphinx-virtualenv
```

Comment:
If this is actually making a difference, it sounds like we are not catching or python is not throwing an appropriate error on a failed dependency?

Reply:
I searched through my `.bash_history` and this seemed to be the original context which brough about the problem:

```console
make users-guide
-- Apparently this failed with some warning that I have to install python3.10-venv, so I did.
apt install python3.10-venv
-- And forgot the sudo ^^
sudo apt install python3.10-venv
-- Then I tried to run make users-guide again
make users-guide
-- That seems to have failed too, so I tried to build the requirements.
make users-guide-requirements 
make .python-sphinx-virtualenv
make users-guide
```

I only came back to cabal the next day, so I don't have to original console output anymore.